test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -375,6 +383,43 @@ describe("GET tests", () => {
     expect(resp.text).toContain(CHECK_YOUR_ANSWERS_PAGE_TRUST_TITLE);
   });
 
+  test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with trust data and feature flag on`, async () => {
+
+    mockIsActiveFeature.mockReturnValueOnce( false ); // another flag
```

Comment:
Probably we need it to bypass the `serviceAvailabilityMiddleware` feature flag, but for sure the `another flag` does not add any value

Reply:
I thought the comment was removes sorry. Can we remove please not useful comments like `// yet another another flag` and` // another flag`

Reply:
This style of using feature flags is super unstable. I did a merge from main and found out I needed to add another call to mockReturnValueOnce.

It is hard to know what feature flag these are but if you know then please say what the two are and I will comment appripriately

Reply:
You are absolutely right and I removed some of them, but as you know we have got many MVP projects that need to be feature flagged. To be honest I love the old school of a development branch instead of working with feature flag, but this will increase pressure on tester when all merged